src/cartesian/getTicks.ts

```diff
@@ -24,9 +24,17 @@ function getTicksEnd(
 
   for (let i = len - 1; i >= 0; i--) {
     let entry = result[i];
-    const size = getTickSize(entry, i);
+    let size: number | undefined;
+    const getSize = () => {
+      if (size === undefined) {
+        size = getTickSize(entry, i);
+      }
+
+      return size;
+    };
```

Comment:
```suggestion
    const getSize = () => getTickSize(entry, i);
```

My question remains: In which case would size already be set? Why is this not just

Reply:
It does, I was not aware. Would it make sense to add this as a comment?

Reply:
I think we can see that from the code; it's pretty local to the lines 89-115. In which line do you think a comment would fit?

Reply:
Totally fair. Not a requirement.